wwwbase/search.php

```diff
@@ -381,7 +381,6 @@
 SmartyWrap::assign('showParadigm', $showParadigm);
 SmartyWrap::assign('locParadigm', session_user_prefers(Preferences::LOC_PARADIGM));
 SmartyWrap::assign('paradigmLink', $paradigmLink);
-SmartyWrap::assign('advancedSearch', $text || $sourceId);
```

Comment:
Ștergerea acestei linii duce la un bug: URL-ul `/definitie-dex09/abstrus`, care arată doar o definiție din DEX 2009, ar trebui să apară cu meniul expandat, indiferent de valoarea preferinței. Dar dacă nu am definit preferința, meniul nu apare. La fel și pentru căutări full-text. Deoarece anumite opțiuni avansate sunt folosite, este important ca ele să fie vizibile.

Ce-ai spune să păstrăm linia într-un `if`:

    if ($text || $sourceId) { // must show the advanced search menu regardless of preference
      SmartyWrap::assign('advancedSearch', true);
    }